slip.js

```diff
@@ -502,7 +502,9 @@ window['Slip'] = (function(){
             this.otherNodes = [];
 
             // selection on iOS interferes with reordering
-            document.addEventListener("selectionchange", this.onSelection, false);
+            if (/Safari/.test(navigator.userAgent) && !/Chrome/.test(navigator.userAgent)) {
```

Comment:
Apparently not only on Mobile: 

https://dev.windows.com/en-us/microsoft-edge/platform/faq/what-is-the-microsoft-edge-user-agent-st/